src/UrlHandlers/TokensApiHandler.php

```diff
@@ -16,10 +17,14 @@ public function __construct(array $childUrlHandlers = [])
     {
         parent::__construct($childUrlHandlers);
 
-        $this->post("/tokens", function($params, WebRequest $request){
+        $this->post("/token", function($params, WebRequest $request){
```

Comment:
is this module not being used anywhere? changing this will break any other platforms using it

Reply:
This is a radical new version of the module - the only other using it is countrysidetagorders so is okay to change at this stage. Again this was to standardise the end point against with other auth apis.